pkg/accounts/kubernetes/validator.go

```diff
@@ -122,33 +131,44 @@ func makeClientFromSecretRef(ctx context.Context, ref *interfaces.SecretInNamesp
 	if err != nil {
 		return nil, err
 	}
-	return config.ClientConfig()
+
+	cfg, err := config.RawConfig()
+	if err != nil {
+		return nil, fmt.Errorf("Error parsing kubeconfigFile:\n  %w\n  kubeconfigFile contents:\n\n%s", err, str)
```

Comment:
Go errors generally start with lowercase. 
'd also not log the kubeconfig content as it may contain tokens.

Reply:
As I write this I realize, this is already done multiple times in that same file. I don't think it's a good idea.